### PR TITLE
Updated `gitleaks` scanner to latest release v.7.6.1

### PR DESCRIPTION
**Pull request by rseedorff**
## Description

This PR if applied updates the `gitleaks` scanner to latest release v.7.6.1.

### Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `npm test` runs for the whole project.
* [x] Make codeclimate checks happy

### CODE REVIEW

File: scanners/gitleaks/scanner/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ARG scannerVersion=latest
 FROM zricethezav/gitleaks:${scannerVersion}
-RUN adduser -S -H -u 1001 gitleaks
+USER root
```
Comment (J12934) [inline scanners/gitleaks/scanner/Dockerfile:7]: Why is this required?

Comment (fuhrmeistery) [reply scanners/gitleaks/scanner/Dockerfile:7]: wrapper.sh is owned by root:root. The rights to execute this script can therefore not be changed as user 'gitleaks' that is now used in the base image

**Review by fuhrmeistery — APPROVED** [review]
